docker/vue-app/src/LEAF_FormEditor_App_vue.js

```diff
@@ -204,24 +276,68 @@ export default {
          * @param {string} content 
          * @returns string with tags removed and remaining characers decoded
          */
-        stripAndDecodeHTML(content='') {
+        stripAndDecodeHTML(content = '') {
             const elDiv = document.createElement('div');
             elDiv.innerHTML = content;
-            const text = XSSHelpers.stripAllTags(elDiv.innerText);
-            return text;
+            return XSSHelpers.stripAllTags(elDiv.innerText);
+        },
+        showLastUpdate(elementID = '', text = '') {
+            const el = document.getElementById(elementID);
+            if(el) {
+                el.innerText = text;
+                el.style.opacity = 1;
+                el.style.border = '2px solid #20a0f0';
+                setTimeout(() => {
+                    el.style.border = '2px solid transparent';
+                }, 750);
+            }
         },
         /**
-         * used to track whether js code and styles for orgchart formats have been downloaded from the nexus
-         * @param {string} selectorType group, employee, position
+         * Sends background call to get more immediate feedback during navigation about login or token status,
+         * since the response from the index.php case is only returned on initial load.
          */
-        addOrgSelector(selectorType = '') {
-            this.orgSelectorClassesAdded[selectorType] = true;
+        setDefaultAjaxResponseMessage() {
+            $.ajax({
+                type: 'POST',
+                url: `ajaxIndex.php?a=checkstatus`,
+                data: {
+                    CSRFToken,
+                },
+                success: (res) => {
+                    this.ajaxResponseMessage = res;
+                },
+                error: (err) => reject(err)
+            });
         },
-        /**
-         * used to force rerender of the form view controller component
-         */
-        toggleIndicatorCountSwitch() {
-            this.indicatorCountSwitch = !this.indicatorCountSwitch;
+        initializeOrgSelector(selType = 'employee', indicatorID = 0, selectorIDPrefix = '', initialValue = '') {
+            const prefix = selType === 'group' ? 'group#' : '#';

```

Comment:
Are we sure selType will be lowercase?

Reply:
It should be, but I added a toLower for the argument just in case.